src/shared/entities.d.ts

```diff
@@ -266,3 +266,6 @@ export type LockedDistricts = readonly boolean[];
 
 export type UintArray = Uint8Array | Uint16Array | Uint32Array;
 export type UintArrays = ReadonlyArray<UintArray>;
+export interface RegionLookupProperties {
+  readonly name?: string;
+}
```

Comment:
We should make this match the type being used in the frontend (which is also more accurate as to what the API can return), and make use of it in `src/client/api.ts` :

```suggestion
export type RegionLookupProperties = Record<string, unknown>;
```